pkg/services/sqlstore/database_wrapper.go

```diff
@@ -0,0 +1,109 @@
+package sqlstore
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"time"
+
+	"github.com/gchaincl/sqlhooks"
+	"github.com/go-sql-driver/mysql"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+	"github.com/lib/pq"
+	"github.com/mattn/go-sqlite3"
+	"github.com/prometheus/client_golang/prometheus"
+	"xorm.io/core"
+)
+
+var (
+	databaseQueryCounter   *prometheus.CounterVec
+	databaseQueryHistogram *prometheus.HistogramVec
+	logger                 log.Logger = log.New("DBMetrics")
```

Comment:
I think we in general use lowercase logger names. We already have a global `sqlstore` logger in this package. Maybe rename this `sqlstore.metrics`? And maybe move into field of `databaseQueryWrapper`?

Reply:
:+1: